packages/TorneloScoresheet/src/components/Toolbar.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from 'react';
+import { Image, StatusBar, StyleSheet, Text, View } from 'react-native';
+import { useAppModeState } from '../context/AppModeStateContext';
+import {
+  colours,
+  ColourType,
+  statusBarStyleForColor,
+  textColour,
+} from '../style/colour';
+import { primary as primaryFont } from '../style/font';
+import { AppMode } from '../types/AppModeState';
+import IconButton from './IconButton';
+import Sheet from './Sheet';
+
+/**
+ * The App's toolbar.
+ *
+ * Depending on what mode the app is in, the way the toolbar looks will be different,
+ * as well as what actions and buttons are available
+ */
+
+const BLACK_LOGO_IMAGE = require('../../assets/images/icon-logo-black-500.png');
+const WHITE_LOGO_IMAGE = require('../../assets/images/icon-logo-white-500.png');
+
+const colourForMode: Record<AppMode, ColourType> = {
+  [AppMode.ArbiterSetup]: colours.secondary,
+  [AppMode.PlayerScoresheetRecording]: colours.primary,
+  [AppMode.ResultDisplay]: colours.primary,
+  [AppMode.TablePairing]: colours.tertiary,
+};
+
+const backgroundColorStyle = (backgroundColor: string) => ({
+  backgroundColor,
+});
+
+const Toolbar: React.FC = () => {
+  const [appModeState] = useAppModeState();
+  const [showSheet, setShowSheet] = useState(false);
+  const handleHelpPress = () => {
+    setShowSheet(a => !a);
+  };
+  const currentColour = colourForMode[appModeState.mode];
+  const currentTextColour = textColour(currentColour);
+  return (
+    <>
+      <StatusBar barStyle={statusBarStyleForColor(currentColour)} />
+      <Sheet
+        title="Help"
+        dismiss={() => setShowSheet(false)}
+        visible={showSheet}
+        content="Put help here..."
```

Comment:
is the sheet component only going to be for instructions on how to use each mode in the app? If so how do we plan to set the content? Should we have a Record<appMode, string> like you did for colour or are we planning on using the sheet for other stuff too?

Reply:
I think the idea at this stage is that we'll have a separate file that has a Record of all of the help strings, like you suggest. This concept shouldn't be tied to the `Sheet` component though, since we may want to use sheets for other things.
So we'll have a file with help strings that the `Toolbar` will import and pass the correct string to the `Sheet`